acceptance-tests/storage_test.go

```diff
@@ -42,4 +42,44 @@ var _ = Describe("Storage", Label("storage"), func() {
 
 		appOne.DELETE(blobName)
 	})
+
+	It("works with Spring and  GCP Cloud libraries", Label("spring"), func() {
+		By("creating a service instance")
+		bucketName := random.Name(random.WithPrefix("csb"))
+		serviceInstance := services.CreateInstance(
+			"csb-google-storage-bucket",
+			"default",
+			services.WithParameters(map[string]any{
+				"name": bucketName,
+			}),
+		)
+		defer serviceInstance.Delete()
+
+		By("pushing the unstarted app twice")
+		appOne := apps.Push(apps.WithApp(apps.JDBCTestApp), apps.WithTestAppManifest(apps.StorageTestAppManifest))
```

Comment:
Using `apps.JDBCTestApp` is a bit confuding to me since this is not using any of the JDBCTestApps. 
Am I missing something?

Reply:
Good catch! It is a mistake. Thank you